index.js

```diff
@@ -124,6 +124,7 @@ function makeCachedResolver (prefix, fieldName, cache, originalFieldResolver, po
   })
   return async function (self, arg, ctx, info) {
     if ((skip && await skip(self, arg, ctx, info)) ||
+      (info.operation && info.operation.operation === 'mutation') ||
```

Comment:
in this way, the "event" `onSkip` is called, but it shouldn't
imo the mutation skip should just call the original resolver

Reply:
yeah I was thinking about that too but wasn't sure if `onSkip` should be called or not

Reply:
I'd go for something like this

```js
if(info.operation && info.operation.operation === 'mutation') {
  return originalFieldResolver(self, arg, ctx, info)
}

if ((skip && await skip(self, arg, ctx, info)) ||
  (policy && policy.skip && await policy.skip(self, arg, ctx, info))) {
  onSkip()
// ...
```